### DISCUSSION TITLE
How to Enable CSS Grid by Default

### DISCUSSION THREAD

**Discussion by HadyShaltout** [twbs/bootstrap#37964] [Q&A]
I need to generate Bootstrap with CSS Grid enabled by running **npm install** so the clients can get the grid without any additional steps.

I'm thinking about including the **enable-cssgrid: true** in my project's **package.json**, Something like that:

```
{
 "name": "PROJECT",
 "description": "DESC",
 "dependencies": {
   "backbone": "^1.4.1",
   "bootstrap": "^5.3.0-alpha1", // Need to add enable-cssgrid variable here!!
   "jquery": "^3.6.3"
  }
}
```

Is that possible?

**Comment by lobap** [comment]
No, I believe it is not possible to enable CSS Grid directly in your project's package.json file. CSS Grid is a feature of CSS itself and is not a separate package that can be installed or enabled through npm.

Bootstrap 5.2 does include the ability to enable or disable CSS Grid layout on a per-build basis by using the enable-grid Sass variable. This means that you can enable or disable CSS Grid when building Bootstrap by modifying this variable in your own Sass setup.

To enable CSS Grid in Bootstrap 5.2, you need to set the enable-grid variable to true in your Sass setup before importing Bootstrap. This will include the CSS Grid layout classes in the compiled CSS, allowing you to use them in your HTML:

`
$enable-grid: true;
`

**Reply by HadyShaltout** [reply]
Thanks bro, 
the bootstrap module will be included after running **npm install** so the sass file will be imported by this command, How to setup the sass variables before importing?

**Comment by blachawk** [comment]
> To enable CSS Grid in Bootstrap 5.2, you need to set the enable-grid variable to true in your Sass setup before importing Bootstrap. 

Yeah that little bit of info would of been nice for the official BS team to add that on[ their official docs](https://getbootstrap.com/docs/5.3/layout/css-grid/l). And that process also works for 5.3
